csatar-theme/pages/tag.htm

```diff
@@ -44,4 +44,14 @@
 
     {% component 'basicForm' %}
 
+    {% if scout.getTeamChangeHistory()|length > 0 %}
+    <div class="field-section toolbar-item toolbar-primary mb-2">
+        <h4 style="display:inline;">{{ 'Előzmények' | _ }}</h4>
+    </div>
+
+            {% for teamChange in scout.getTeamChangeHistory() %}
```

Comment:
Nem lenne jobb scout.getTeamChangeHistory()-t csak egyszer lekerni, s eltarolni? A 47. sorban is le van kerve. Csak azert, hogy ne allitsa 2x ossze a tombot.

Reply:
De, javítva